redhat/apmplanner2-0.patch

```diff
@@ -0,0 +1,63 @@
+diff --git a/apm_planner.pro b/apm_planner.pro
```

Comment:
Kinda RedHat approach. Keeping original source as is and issuing all distro-specific changes as patches. Thus I did so.